src/css/zooniverse-react-components.styl

```diff
@@ -6,3 +6,7 @@
 @require './paginator'
 @require './zoo-footer'
 @require './admin-layout-indicator'
+@require './tutorial'
```

Comment:
Just a heads up I also changed the import to import using a wildcard in add-header branch. Let's merge that branch first and rebase this branch off those organizational changes.